io.catenax.shared.business_partner_certificate/1.0.0/BusinessPartnerCertificate.ttl

```diff
@@ -0,0 +1,171 @@
+    samm:description "Version of the certificate as defined on the document, usually the specific version of a certification standard"@en;
+    samm:characteristic samm-c:Text ;
+    samm:exampleValue "2015". 
+  
+ :registrationNumber a samm:Property;
+    samm:preferredName "Certificate Registration Number"@en;
+    samm:description "Registration number of the certificate as in the certification document"@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "12 198 54182 TMS".
+
+
+  :areaOfApplication a samm:Property;
+    samm:preferredName "area of application"@en;
+    samm:description "Details on which areas / application types a certification is valid "@en;
+    samm:characteristic samm-c:Text;
+    samm:exampleValue "Development, Marketing und Sales and also Procurement for interiour components".
+ 
+  
+ :enclosedSites a samm:Property;
+    samm:preferredName "Enclosed Sites"@en;
+    samm:description "Optional enclosed sites of the legal entity which are enclosed by the certification "@en;
+	samm:characteristic  [ a samm-c:List ; samm:dataType :enclosedSiteEntity].
+	
+	  
+ :enclosedSiteEntity a samm:Entity;
+    samm:preferredName "Defines enclosed sites "@en;
+    samm:description "Optional enclosed sites where certificatation is valid including details about area of application"@en;
+    samm:properties (:enclosedSiteBPN [samm:property :areaOfApplication;samm:optional true]).
+
+  :enclosedSiteBPN a samm:Property ;
+   samm:preferredName "enclosedSiteBPN"@en ;
+   samm:description "The BPN of an enclosed site "@en ;
+   samm:characteristic ext-number:BpnsTrait ;
+   samm:exampleValue "BPNS00000003AYRE".
+
+ 
+  :validFrom a samm:Property;
+    samm:preferredName "valid From"@en;
+    samm:description "certificate valid from as in the certification document."@en;
+    samm:characteristic :Date;
+    samm:exampleValue "2023-01-25"^^xsd:date.
+
+ :validUntil a samm:Property;
+    samm:preferredName "Certificate Valid Until"@en;
+    samm:description "Certificate valid until as in the certification document. If certificate never expires value until expected to be 9999-12-31"@en;
+    samm:characteristic :Date;
+	samm:exampleValue "2026-01-24"^^xsd:date.
+    
+:Date a samm:Characteristic ;
+   samm:preferredName "Date"@en ;
+   samm:description "Describes a property which contains the date in english format."@en ;
+   samm:dataType xsd:date;
+   samm:exampleValue "2026-01-24"^^xsd:date.
+  
+:validator a samm:Property ;
+   samm:preferredName "validator"@en ;
+   samm:description "The BPN of the data service provider who validate the given certificate"@en ;
+   samm:characteristic :trustValidatorCharacteristic.
+
+:trustValidatorCharacteristic a samm:Characteristic;
+	samm:preferredName "validitor Characteristic"@en;
+	samm:description "The BPN of the data service provider who validate the given certificate"@en ;
+    samm:dataType :trustValidatorEntitity . 
+   
+   
+:trustValidatorEntitity a samm:Entity ;

```

Comment:
typo in property name `trustValidatorEntitity`. please correct this and references.

Reply:
Fixed typo